views/pages/GraphPage.jsx

```diff
@@ -500,12 +463,10 @@ class Graph extends Component {
       })
       .then((res) => {
         if (res.status === 201) {
-          window.location.replace(
-            routes.subjectView(
-              this.state.subject.project,
-              this.state.subject.sid
-            )
-          )
+          this.setState({
+            preferences: { ...this.state.preferences, config: configurationId },
+          })
+          this.fetchGraph(this.state.subject.project, this.state.subject.sid)
```

Comment:
we call fetch graph using data from state and from props. Can we do one or the other?